content/applications/inventory_and_mrp/purchase/manage_deals/blanket_orders.rst

```diff
@@ -1,204 +1,183 @@
+:guilabel:`Quantities` fields can be edited. Doing so sets how new quotations should be populated
+when using this purchase agreement.
 
 .. image:: blanket_orders/blanket-orders-edit-agreement-type.png
    :align: center
    :alt: Purchase Agreement type edit screen for blanket orders.
 
 There are two options that can be activated for :guilabel:`Lines`:
 
-- :guilabel:`Use lines of agreement`: when creating a new quotation, the product lines will
-  pre-populate with the same products listed on the blanket order, if said blanket order is chosen
-  on the new quotation.
-- :guilabel:`Do not create RfQ lines automatically`: when creating a new quotation and selecting
-  an existing blanket order, the settings will carry over to the new quotation, but the product
-  lines will not populate.
+- :guilabel:`Use lines of agreement`: when creating a new quotation, the product lines pre-populate
+  with the same products listed on the blanket order, if said blanket order is chosen for the new
+  quotation.
+- :guilabel:`Do not create RfQ lines automatically`: when creating a new quotation, **and**
+  selecting an existing blanket order, the settings will carry over to the new quotation, but the
+  product lines will **not** populate.
 
 And, there are two options that can be activated for :guilabel:`Quantities`:
 
 - :guilabel:`Use quantities of agreement`: when creating a new quotation, the product quantities
-  listed on the blanket order will pre-populate on the product lines, if said blanket order is
-  chosen on the new quotation.
-- :guilabel:`Set quantities manually`: when creating a new quotation and selecting an existing
-  blanket order, the product lines will pre-populate, but all quantities will be set to **0**. The
-  quantities will need to be manually set by the user.
-
-Once the desired changes have been made (if any), click :guilabel:`New` (via the breadcrumbs, at
-the top of the page) to navigate back to the blanket order form, and click :guilabel:`Confirm` to
-save this new purchase agreement. Once confirmed, the blanket order changes from *Draft* to
-*Ongoing*, meaning this agreement can be selected and used when creating new :abbr:`RFQs (Requests
-for Quotation)`.
+  listed on the blanket order pre-populate on the product lines, if said blanket order is chosen for
+  the new quotation.
+- :guilabel:`Set quantities manually`: when creating a new quotation, **and** selecting an existing
+  blanket order, the product lines pre-populate, but all quantities will be set to **0**. The
+  quantities **must** be manually set by the user.
+
+Once any desired changes have been made, click :guilabel:`New` (via the breadcrumbs, at the top of
+the page) to navigate back to the blanket order form. Then, click :guilabel:`Confirm` to save this
+new purchase agreement.
+
+Once confirmed, the blanket order's stage (in the upper-right corner) changes from *Draft* to
+*Ongoing*, meaning this agreement can be selected and used when creating new |RfQs|.
```

Comment:
GUIs should be applied to 'Draft' and 'Ongoing' because they appear on the UI

Reply:
Suggestion implemented ✅